user/serializers.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+from user.models import User
```

Comment:
The import of `User` from `user.models` is unnecessary since you are using `get_user_model()` to reference the user model. You can safely remove this import.